x/observer/migrations/v7/migrate.go

```diff
@@ -29,16 +29,24 @@ func MigratePolicies(ctx sdk.Context, observerKeeper observerKeeper) error {
 	for _, adminPolicy := range params.AdminPolicy {
 		if adminPolicy != nil {
 
-			// Convert group1 -> emergency and group2 -> admin
-			policyType := authoritytypes.PolicyType_groupAdmin
 			if adminPolicy.PolicyType == types.Policy_Type_group1 {
-				policyType = authoritytypes.PolicyType_groupEmergency
+				// for policy group 1, we set the policy type to emergency
```

Comment:
So even after this mightation observers.AdminPolicy would exist . Would that get removed in the next release?

Reply:
Yes, we are removing all params in any case for v47 upgrade